optimum/neuron/utils/generation_utils.py

```diff
@@ -0,0 +1,188 @@
+# coding=utf-8
```

Comment:
I'd rather not add yet another util file here: I think all these functions can go into the same `generation/utils.py` file.

Also, I would prefix them with an underscore, since they are never used anywhere else.

Reply:
Yes I agree, I'll move the functions in this class to `generation/utils.py`